### DISCUSSION TITLE
Removed station but Shoutcast is already online and works

### DISCUSSION THREAD

**Discussion by heyDaino** [AzuraCast/AzuraCast#5288] [Q&A] [answered]
Hello.
I remove a station via Azuracast panel but the Shoutcast port is already online and works.
I tried to restart the server, I tried to recreate the station on the same port (it works but sc_serv info is not updated because the previous passwords work) but nothing... it continues go on! 

there is a command via server to kill that specific port of shoutcast?

I wanted to mention that this problem only happened once, with the other stations there are none

**Comment by Vaalyn** [comment, marked as answer]
You might be encountering [this](https://github.com/AzuraCast/AzuraCast/issues/4898) known bug.

To find out if this is such a "phantom station" scenario and how to fix it please follow these steps:
- Enter the Docker container via `./docker.sh bash`
- Then when you are inside the container run `sudo supervisorctl status` to get a list of the running frontend and backend processes that are managed by the supervisord in AzuraCast
- If you can see a station in that list that does not exist anymore in the web interface take note of the number shown in the output of the previous command
  - Be careful to really get the correct number here, check with the list on the Dashboar in the `Station Overview` by hovering over the "Manage" button of each station with your mouse pointer and looking at the number in the URL shown in the bottom left corner of your browser (see screenshot)
  - Example output: `station_13:station_13_frontend`
  - Number: `13`
- Shutdown the frontend and backend processes of the phantom station via `sudo supervisorctl stop station_13:station_13_frontend` and `sudo supervisorctl stop station_13:station_13_backend`
  - Replace the `13` with the number you got from the previous step
- Go into the stations directory via `cd /var/azuracast/stations`
- List all station directories via `ls -la`
- Look for the directory of the phantom station
  - Example station directory: `azura_test`
- Remove the phantom stations directory via `sudo rm -rf /var/azuracast/stations/azura_test`
  - Be careful to not delete the wrong stations directory

<img width="1406" alt="image" src="https://user-images.githubusercontent.com/13745863/163031273-8bb4c1a8-c7a1-4fe6-a005-728d03ad722d.png">
